index.html

```diff
@@ -1,2 +1,144 @@
+        </p>
+        <p>
+           We have a deep respect for open source work - we use it throughout our products and tools.  One of the best ways we could give back is to provide best-in-breed security, tools, and applications back in open source form.
+        </p>
+        <p><a class="btn btn-warning btn-lg" href="http://www.rapid7.com/company/impact.jsp" role="button">Learn more about Rapid7's commitment to the security community &raquo;</a></p>
+      </div>
+    </div>
+    </div> <!-- /col-md-2 navigation -->
+      <div class="container-fluid" id="security">
+        <div class="row">
+          <div class="col-md-8 col-md-offset-2 grow" style="border: 10px solid #FF7F00;">
+            <h2><i class="fa fa-shield"></i> Security related</h2>
+            <h3 style="color:#FF7F00">Take advantage of these tools to help secure your environment.</h3>
+            <p  class="cat-desc">Here in this section, you'll find all the open source things we've created to support
+              our various security initatives.  From the many, many <a href="https://github.com/rapid7/metasploit-framework">metasploit-framework</a> projects, to things like <a href="https://github.com/rapid7/awsaml">awsaml</a> - which we use to manage AWS key rotation for our CLI development work.</p>
+              <p>For a full list of ALL the Metasploit components available to you, there's a <a href="https://github.com/rapid7/hackazon">dedicated page</a>.</p>
+            <h4>Metasploit Related</h4>
+            <ol>
+              <li><a href="https://github.com/rapid7/metasploit-framework">metasploit-framework</a></li>
+              <li><a href="https://github.com/rapid7/metasploit_data_models">metasploit_data_models</a></li>
+              <li><a href="https://github.com/rapid7/metasploit-payloads">metasploit-payloads</a></li>
+              <li><a href="https://github.com/rapid7/hackazon">hackazon</a></li>
+              <li><a href="link here to another page dedicated to Metasploit bits">More Metasploit projects here!</a></li>
+            </ol>
+            <h4>Utilities</h4>
+            <ol>
+            <li><a href="https://github.com/rapid7/awsaml">awsaml</a></li>
+            <li><a href="https://github.com/rapid7/guardian">guardian</a></li>
+            </ol>          
+          </div>
+        </div>
+      </div>
+      <div class="container-fluid" id="config">
+        <!-- Example row of columns -->
+        <div class="row">
+          <div class="col-md-8 col-md-offset-2 grow" style="border: 10px solid #497CE2;">
+            <h2><i class="fa fa-cogs"></i> Configuration Management</h2>
+            <h3 style="color:#497CE2">Something clever about config management.</h3>
```

Comment:
Is this clever enough? "Leverage these tools to make live updates to running servers easier."